tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -1181,6 +1181,9 @@ class CudnnRnnDescriptor : public dnn::RnnDescriptor {
     }
 
     // Create the params handle.
+    // TODO(kaixih): Should be removed when cudnnRNNForward*** and
```

Comment:
Maybe add a `TODO(kaixih@nvidia.com)`?  Otherwise by default folks would assume we're talking about an ex-Googler.

Reply:
Done.